task20/js/utils/createElement.js

```diff
@@ -0,0 +1,6 @@
+export const createElementWithClassName = (element, name) => {
+    const createdElement = document.createElement(element);
+    createdElement.classList.add(name);
+    return createdElement;
+  };
```

Comment:
[comment]
I thought it was nice that you separated the files into utils and modules!👏

Reply:
Thank you for your kind comments.:)